junit-jupiter-api/src/main/java/org/junit/jupiter/api/IndicativeSentencesGeneration.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2015-2019 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.jupiter.api;
+
+import static org.apiguardian.api.API.Status.EXPERIMENTAL;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.apiguardian.api.API;
+
+/**
+ * {@code @IndicativeSentencesGeneration} is used to declare a custom separator
+ * by {@code IndicativeSentences}, if this notation was not declared
+ * will be use ", " as separator by default.
+ *
+ * @since 5.6
+ * @see DisplayNameGenerator
+ */
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@Inherited
+@API(status = EXPERIMENTAL, since = "5.6")
+public @interface IndicativeSentencesGeneration {
```

Comment:
I'm wondering if we could add `@DisplayNameGeneration(DisplayNameGenerator.IndicativeSentences.class)` here, so that it could be omitted on the class? Would it make sense?

Reply:
Do you mean that the whole implementation should be on that file?

Reply:
That's also what I meant in https://github.com/junit-team/junit5/pull/2124#discussion_r359873187. If we annotate `IndicativeSentencesGeneration` with `@DisplayNameGeneration(DisplayNameGenerator.IndicativeSentences.class)`, users could use `@IndicativeSentencesGeneration` without having to also annotate their classes with `@DisplayNameGeneration(DisplayNameGenerator.IndicativeSentences.class)`.

Reply:
Yeah, now I can see what you mean, I've never heard of Meta-Annotations, luckily, this page
https://dzone.com/articles/what-are-meta-annotations-in-java had a very good example that explained to me what I wasn't picking up. 😅